guides/framework-guides/framework-mjml.mdx

```diff
@@ -151,15 +151,15 @@ export const mjmlTemplate = (inputs: Inputs) => {
 
 3. And as final step, we need to define the workflow that uses the template defined above.
 ```jsx
-import { Echo } from "@novu/echo";
+import { Client, workflow } from "@novu/framework";
 import { mjmlTemplate } from "./mjml";
 
-export const echo = new Echo({
+export const client = new Client({
```

Comment:
The client instantiation can be removed if we do this check in the SDK. That is `strictAuthentication` should be `false` is `process.env.NODE_ENV === "development"`.

We also need to inform the customers that if NODE_ENV is not set, strictAuthentication is enabled. 

cc: @rifont can we add this to Linear?